src/core/portal/portal-directives.ts

```diff
@@ -0,0 +1,110 @@
+import {Portal} from './portal';
```

Comment:
The general best practice is to organize code by feature / area than by type of thing it is (directive, service, etc.). As such, all the portal code should go together. This would be part of the general component-building toolkit.